racunaljka.js

```diff
@@ -0,0 +1,131 @@
+var writes = document.getElementById('screen');
```

Comment:
Please investigate this approach a bit more, until you fully understand it. Once you do, apply it in this pull request.